console-subscriber/src/aggregator.rs

```diff
@@ -59,6 +59,9 @@ pub(crate) struct Aggregator {
 
     /// Map of task IDs to task stats.
     stats: TaskData<Stats>,
+
+    /// The time "state" of the aggregator, such as paused or live.
+    temporality: Temporality,
```

Comment:
Having one temporality for the aggregator seems like a potential issue. Since multiple consoles can connect to the same application, if one user pauses their console, sending the pause command to the application, all consoles will be paused. Similarly, if I unpause my console, everyone else's console unpauses, too. It doesn't _seem_ like this is the behavior we want when multiple consoles are connected?

If we're going to implement pausing in the aggregator-side (rather than in the UI-side), it seems like we would want to track a separate temporality for each client connection, and decide whether we are going to send updates to each client independently?